docs/install.rst

```diff
@@ -1,103 +1,134 @@
-<https://help.github.com/articles/adding-a-remote/>`_ and checking out a
-branch.  You can combine changes from multiple users by merging them into a
-branch in your local version of the repository.
+The tables below list several packages that provide OPS with extra
+functionality when installed, and include links to their ``conda`` packages.
 
-Additional functionality of the ``conda_ops_dev_install.sh`` script is
-described in :ref:`quick-dev-install`.
+Molecular dynamics engines
+--------------------------
 
-.. _manual-install:
++------------------------+-------------------------------------------------+
+| | **OpenMM**           |                                                 |
+| | |openmm-conda|_      |                                                 |
++------------------------+-------------------------------------------------+
+| | **OpenMMTools**      | OPS has special support for integrators from    |
+| | |openmmtools-conda|_ | OpenMMTools (and we strongly recommend using    |
+|                        | one of the reversible integrators provided      |
+|                        | there, such as VVVR).                           |
++------------------------+-------------------------------------------------+
+| | **GROMACS**          | OPS will work with any installed GROMACS; it    |
+| | |gromacs-conda|_     | does not need to come from ``conda``.           |
++------------------------+-------------------------------------------------+
 
-Manual Installation
-===================
+.. |openmm-conda| image:: https://img.shields.io/conda/vn/conda-forge/openmm
+.. |openmmtools-conda| image:: https://img.shields.io/conda/vn/conda-forge/openmmtools
+.. |gromacs-conda| image:: https://img.shields.io/conda/vn/bioconda/gromacs
 
-If you don't want to use ``conda``, you will have to manually obtain the
-dependencies, which you can see listed under ``install_requires`` in
-``setup.py``. Then you can install from our GitHub repository.
+.. _openmm-conda: https://anaconda.org/conda-forge/openmm
+.. _openmmtools-conda: https://anaconda.org/conda-forge/openmmtools
+.. _gromacs-conda: https://anaconda.org/bioconda/gromacs
 
-Clone the source code repository from GitHub::
+Collective variables
+--------------------
 
-  $ git clone https://github.com/openpathsampling/openpathsampling.git
+In addition to the optional packages listed below, OPS ships with MDTraj
+built-in.
 
-Then, in the directory containing the source code, you can install it with ::
++------------------------+-------------------------------------------------+
+| | **PyEMMA**           |                                                 |
+| | |pyemma-conda|_      |                                                 |
++------------------------+-------------------------------------------------+
+| | **py-PLUMED**        |                                                 |
+| | |plumed-conda|_      |                                                 |
```

Comment:
is this just a python wrapper or does it include an actual install of plumed as well (I trul

Reply:
py-plumed [recipe requires plumed](https://github.com/conda-forge/py-plumed-feedstock/blob/278ece433668b8128a9db0d73832daad2264d02a/recipe/meta.yaml#L30) (which has a separate [feedstock](https://github.com/conda-forge/plumed-feedstock/)), so installing from conda brings them both. Other installations of the Python wrapper may require separate install of PLUMED (for example, if you get it from [PyPI](https://pypi.org/project/plumed/), where the requirement is documented pretty clearly.)

Since the Python wrappers allow you to provide the PLUMED kernel at runtime there's no reason not to install with conda -- if you have a custom build of PLUMED, you can still use the conda Python wrappers.